packages/js/email-editor/src/components/template-select/select-modal.tsx

```diff
@@ -1,8 +1,15 @@
+/**
+ * WordPress dependencies
+ */
 import { useState, useEffect, memo } from '@wordpress/element';
 import { store as editorStore } from '@wordpress/editor';
 import { dispatch } from '@wordpress/data';
 import { Modal, Button, Flex, FlexItem } from '@wordpress/components';
 import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
```

Comment:
Do we really need those comments? It seems quite redundant to me.

Reply:
I added those to a couple of components. Woo and Gutenberg do the same. AFAIK in Gutenberg it is enforced. IMO, it makes the imports visually more organized. But if you don't like them, I can remove them.

Reply:
> If this is not enforced by any tool I wonder if we will be able to maintain those sections.
True. I'll keep it for now. I added a discussion point to our team meeting agenda. We can add the check or remove it...